cypress/cypress/integration/record.spec.ts

```diff
@@ -557,11 +587,85 @@ describe("Record", () => {
     cy.get("[data-cy=video-recorder]").should("not.exist");
   });
 
-  it("shows video recorder if mentor type is VIDEO", () => {
+  it("shows video recorder if mentor type is VIDEO and no video", () => {
     cySetup(cy);
     cyMockDefault(cy, { mentor: mentor2 });
     cy.visit("/record");
     cy.get("[data-cy=video-recorder]").should("exist");
+    // video recorder showing
+    cy.get("[data-cy=video-recorder]").should("have.css", "visibility", "visible");
```

Comment:
does `should("be.visible")` not work in these cases? If we can use it, let's prefer it over css checks